htdocs/core/class/html.formticket.class.php

```diff
@@ -753,7 +755,7 @@ public function selectTypesTickets($selected = '', $htmlname = 'tickettype', $fi
 					print ' selected="selected"';
 				} elseif (in_array($id, $selected)) {
 					print ' selected="selected"';
-				} elseif ($arraytypes['use_default'] == "1" && empty($selected)) {
+				} elseif ($arraytypes['use_default'] == "1" && empty($selected) && !$empty) {
 					print ' selected="selected"';
```

Comment:
Maybe the test !$empty should be replace with a specific list mode using multiselect